controllers/space/space_controller_test.go

```diff
@@ -660,7 +659,8 @@ func TestUpdateSpaceTier(t *testing.T) {
 			spacetest.WithTierName(otherTier.Name), // assume that at this point, the `TemplateTierHash` label was already removed by the ChangeTierRequestController
 			spacetest.WithSpecTargetCluster("member-1"),
 			spacetest.WithStatusTargetCluster("member-1"), // already provisioned on a target cluster
-			spacetest.WithFinalizer())
+			spacetest.WithFinalizer(),
+			spacetest.WithCondition(spacetest.Ready()))
 		hostClient := test.NewFakeClient(t, s, basicTier, otherTier)
 		nstmplSet := nstemplatetsettest.NewNSTemplateSet("oddity", nstemplatetsettest.WithReferencesFor(basicTier), nstemplatetsettest.WithReadyCondition())
 		member1Client := test.NewFakeClient(t, nstmplSet)
```

Comment:
in this promotion case, we could set explicitly
```
ctrl.LastExecutedUpdate = time.Now()
```
to verify that it will ignore the postpone logic

Reply:
I believe we should rather set it to something like 
```
ctrl.LastExecutedUpdate = time.Now().Add(-1 * time.Minute)
```
to emphasize that the last update occurred a long time ago enough (ie, more than the `postponeDelay` duration)

Reply:
see 58f2392